ReportSimulationOutput/tests/output_report_test.rb

```diff
@@ -976,6 +1026,59 @@ def test_timeseries_local_time_dst_and_utc
     assert_equal(1, _check_for_constant_timeseries_step(timeseries_cols[2]))
   end
 
+  def test_timeseries_user_defined_standard_output_variables
+    args_hash = { 'hpxml_path' => '../workflow/sample_files/base.xml',
+                  'timeseries_frequency' => 'hourly',
+                  'include_timeseries_fuel_consumptions' => false,
+                  'include_timeseries_end_use_consumptions' => false,
+                  'include_timeseries_emissions' => false,
+                  'include_timeseries_hot_water_uses' => false,
+                  'include_timeseries_total_loads' => false,
+                  'include_timeseries_component_loads' => false,
+                  'include_timeseries_zone_temperatures' => false,
+                  'include_timeseries_airflows' => false,
+                  'include_timeseries_weather' => false,
+                  'user_output_variables' => 'Zone People Occupant Count, Zone People Total Heating Energy' }
```

Comment:
Can you add a test (or update this one) to include an invalid output variable request (like "Foo")? I think we'd like the measure to still run successfully, but without this column in the timeseries output. Even better would be if we can register a warning for each output variable that is invalid (i.e., doesn't produce EnergyPlus output) and check for the warning in the test.